### PR TITLE
Document the last rebalance state

### PR DESCRIPTION
**Pull request by esproul**
Describe the process that each node will go through and what an
operator should see in the state output.

### CODE REVIEW

File: api/rebalance-state.md [hunk 1]
```diff
@@ -24,8 +24,14 @@ GET
    * `TOPO_REBALANCE_REHASH` (relocating data)
    * `TOPO_REBALANCE_REHASH_VOTE` (waiting for data relocation completion on all nodes)
    * `TOPO_REBALANCE_CLEANUP` (removing data from old topology)
-   * `TOPO_REBALANCE_COMPLETE` (local operations complete, waiting for other
-     nodes to finish before switching topology)
+   * `TOPO_REBALANCE_COMPLETE` (local operations complete)
+   * `TOPO_REBALANCE_COMPLETE_VOTE` (waiting for all other nodes to complete
```
Comment (bincircon) [inline api/rebalance-state.md]: waiting for all others nodes to complete/switch to next topology

Comment (bincircon) [reply api/rebalance-state.md]: it looks like you get cover in the next statement

Comment (esproul) [reply api/rebalance-state.md]: I was a little confused about this, honestly.  I thought _COMPLETE_VOTE was where the actual topology switch occurred locally, or does a node only move into _COMPLETE_VOTE _after_ switching topology itself, locally?

Comment (esproul) [reply api/rebalance-state.md]: Looked at the code, it seems we actually switch topo _before_ setting our state to TOPO_REBALANCE_COMPLETE_VOTE, so I made one more change to the wording.

**Pull request merged**